suite-native/discovery/src/discoveryMiddleware.ts

```diff
@@ -14,7 +14,7 @@ export const prepareDiscoveryMiddleware = createMiddlewareWithExtraDeps(
         // On successful authorization, create discovery instance and run.
         if (deviceActions.authDevice.match(action) && device) {
             dispatch(
-                createAndStartDiscoveryThunk({
+                startDescriptorPreloadedDiscovery({
```

Comment:
Please, use the thunk suffix for newly created redux thunk actions

Reply:
fixed in b739e356c3